daemon/daemon.go

```diff
@@ -392,13 +407,15 @@ func (daemon *Daemon) restore() error {
 				}
 
 				if !alive {
+					logger(c).Debug("Setting stopped state")
 					c.Lock()
 					c.SetStopped(&container.ExitStatus{ExitCode: int(ec), ExitedAt: exitedAt})
 					daemon.Cleanup(c)
 					if err := c.CheckpointTo(daemon.containersReplica); err != nil {
 						log.WithError(err).Error("failed to update stopped container state")
 					}
 					c.Unlock()
+					logger(c).Debug("Set stopped state")
```

Comment:
This should probably be in an `else`, otherwise this is also logged if we failed to update state.

```suggestion
					logger(c).Debug("set stopped state")
```

Also; do we still need this log, or was it mostly used while debugging this issue?